README.md

```diff
@@ -146,7 +152,7 @@ To remove the Docker container and images (this will **delete everything** in th
 
 -   [Usage Tracking](./src/Tracking/README.md)
 
-<p align="center">
+<p style="text-align: center">
```

Comment:
`style` seems to be removed by GitHub 
![image](https://user-images.githubusercontent.com/17435/142198872-1d3a8440-321b-4156-9e27-9a8bae6cafcc.png)
So I think we need to stick to deprecated `align`

Reply:
Nice catch @tomalec!

It was working in my IDE built-in Markdown and confidently didn't check GitHub UI properly.  (my bad)

I rolled back to the previous version, let me know if the PR needs more improvements